docker/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # You can also pull the image manually: docker pull keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/library/ruby:3.2.5-alpine3.20
 
 # check for latest version https://hub.docker.com/_/ruby/tags
-FROM keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/library/ruby:3.2.6-alpine3.20 AS base 
+FROM keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/library/ruby:3.2.7-alpine3.20 AS base 
```

Comment:
Why not update to 3.2.7-alpine3.21 ? In our experience Alpine 3.20 to 3.21 went really smooth.

Reply:
ok, thank you for the hint :-) I will do a local test build